src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingSymbolTranslator.cs

```diff
@@ -184,6 +184,15 @@ private NamedTypeSymbol RetargetNamedTypeDefinition(NamedTypeSymbol type, Retarg
             {
                 Debug.Assert(type.IsDefinition);
 
+                if (type.IsNativeIntegerType)
+                {
+                    if (!SymbolMap.TryGetValue(type, out var retargeted))
+                    {
+                        retargeted = SymbolMap.GetOrAdd(type, Retarget(type.NativeIntegerUnderlyingType, options).AsNativeInteger());
```

Comment:
>Retarget [](start = 62, length = 8)

Also, do we really need to fall back to ```Retarget``` here? It looks like first thing it does is calling ```RetargetNamedTypeDefinition```. Why the indirection?